### DISCUSSION TITLE
Integration issue with Shopify after username change

### DISCUSSION THREAD

**Discussion by kosdigitalexpert** [community/community#42079] [Projects and Issues]
### Select Topic Area

Bug

### Body

Hi there,

I had Github account connected to Shopify and all was working fine. However, I needed to change my username and once I have changed it in Github, it still shows me old username in Shopify. But the main issue is that none of my repositories are shown anymore in Shopify, so I can't use it. I have tried to log out and log in again, but it doesn't show, still shows old username and no repositories. Any ideas?

<img width="1005" alt="1-Screenshot at Dec 18 22-02-49" src="https://user-images.githubusercontent.com/40201109/208321734-724b8740-e699-4a21-b1b3-388f9aae8ece.png">

**Comment by Mim-Jones** [comment]
I have the same issue, did you find a solution @kosdigitalexpert?
